frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -294,20 +293,15 @@ const ManageHostsPage = ({
   const canEnrollHosts =
     isGlobalAdmin || isGlobalMaintainer || isTeamAdmin || isTeamMaintainer;
   const canEnrollGlobalHosts = isGlobalAdmin || isGlobalMaintainer;
-  const canAddNewLabels = isGlobalAdmin || isGlobalMaintainer;
+  const canAddNewLabels = (isGlobalAdmin || isGlobalMaintainer) ?? false;
```

Comment:
I don't think the nullish coalescing is necessary here. Should be the same result either way, unless I'm missing something?

Reply:
this is done for better typing.  both `isGlobalAdmin` and `isGlobalMaintainer` can be a `boolean | undefined` at this moment. if both are undefined than `canAddNewLabels` will be undefined and then i'd have to type check that where it is used in `LabelFilterSelect` component. I'd rather not have to do a type check so I just set it to false so `canAddNewLabels` will always be a boolean.